features/step_definitions/pagination_steps.rb

```diff
@@ -5,3 +5,11 @@
 Then /^I should see pagination with (\d+) pages$/ do |count|
   expect(page).to have_css '.pagination span.page', count: count
 end
+
+Then /^I should see the pagination "Next" link/ do
+  expect(page).to have_css 'a', text: 'Next ›'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.